source/adminguide/storage.rst

```diff
@@ -957,6 +957,9 @@ incremental backups are supported, every N backup is a full backup.
 | Support incremental backup   | No               | Yes              | No  |
 +------------------------------+------------------+------------------+-----+
 
+   .. note::
+      Between versions 4.17.x, 4.18.0 and 4.18.1, KVM volume snapshot backups were not full snapshots and they rely on the snapshots on the primary storage. To prevent any loss of data, care must be taken during revert operation and it must be ensured that the source primary storage snapshot file is present.
+
```

Comment:
@shwstppr, would not it be useful to guide users on how to identify the affected volumes/snapshots, vide https://github.com/apache/cloudstack/issues/8034#issuecomment-1750699975?

Reply:
@GutoVeronezi good idea. Though I'm not sure how to structure it.
Would you mind adding a change in the PR as you must be having better understanding on the process?

Reply:
I missed the last comment; I will put it on my tasks :+1:

Reply:
@GutoVeronezi any update or should we merge this?

Reply:
@shwstppr may be we can continue merging this and handle that steps in a different PR. @GutoVeronezi can you please raise a separate PR for that.